integration_test/third_party_apps_data/applications/jetty/metadata.yaml

```diff
@@ -4,12 +4,9 @@ short_name: Jetty
 long_name: Jetty
 logo_path: /stackdriver/images/jetty.png # supplied by google technical writer
 description: |-
-  The Jetty integration collects
-  access logs. Access logs are parsed into a JSON payload
-  focused on request details. 
-
-  For more information about Jetty, see
```

Comment:
This `For more information...` part is no longer needed because the `app_url` part has been automated to generate this section.